lib/ansible/modules/database/postgresql/postgresql_idx.py

```diff
@@ -506,6 +530,12 @@ def main():
     if concurrent and cascade:
         module.fail_json(msg="Cuncurrent mode and cascade parameters are mutually exclusive")
 
+    if concurrent and unique:
```

Comment:
Postgres allows creating unique indexes concurrently, so this condition isn't needed.

Reply:
:thinking: for some unknown reason I putted this check, thanks again for your review, I removed it